frontend/packages/kubevirt-plugin/src/components/create-vm/dev-console/dev-console-create-vm-form.tsx

```diff
@@ -13,6 +13,7 @@ import { isEmpty } from 'lodash';
 import Helmet from 'react-helmet';
 import { Trans, useTranslation } from 'react-i18next';
 import { RouteComponentProps } from 'react-router';
+import { isCatalogTypeEnabled } from '@console/dev-console/src/utils/catalog-utils';
```

Comment:
We should not add a dependency from the `kubevirt-plugin` to `dev-console`.

Please move the utils from `@console/dev-console/src/utils/catalog-utils` to `@console/shared/src/components/catalog/utils/catalog-utils`.